README.md

```diff
@@ -87,6 +87,7 @@ A curated list of amazingly awesome open source resources for broadcasters inspi
 
 * [FFmpeg](http://ffmpeg.org) - A complete, cross-platform solution to record, convert and stream audio and video.
 * [FFmbc](https://code.google.com/p/ffmbc/) - FFmpeg customized for broadcast and professional usage.
+* [L-SMASH](https://github.com/l-smash/l-smash/) - A rigidly spec-compliant ISOMBFF library, which has full DASH muxing support.
```

Comment:
Hi,

Thank you for the contribution, could you please just fix ISOMBFF to ISOBMFF ?

Cheers,

Michael